docs/pages/cloud/faq.mdx

```diff
@@ -50,14 +52,14 @@ After [connecting](#how-can-i-access-the-tctl-admin-tool) `tctl` to Teleport Clo
 [dynamic tokens](../admin-guide.mdx#short-lived-dynamic-tokens):
 
 ```bash
-$ tctl nodes add --ttl=5m --roles=node,proxy --token=$(uuid)
+tctl nodes add --ttl=5m --roles=node,proxy --token=$(uuid)
 ```
 
 ### When will a security audit be available?
 
 The Teleport Cloud security audit starts in January.
```

Comment:
The audit has been completed on April 12th:

https://goteleport.com/resources/audits/teleport-cloud-security-audit-2021/

Reply:
Ticket tracking this: https://github.com/gravitational/web/issues/1893